src/conv_transpose2d.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
+
+import {Tensor} from './lib/tensor.js';
+import {validateConv2dParams} from './lib/validate-input.js';
+import {computePaddingForAutoPad} from './lib/compute-padding.js';
+import {transpose} from './transpose.js';
+
+/**
+ * Compute a 2-D transposed convolution given 4-D input and filter tensors.
+ * @param {Tensor} input
+ * @param {Tensor} filter
+ * @param {MLConvTranspose2dOptions} options
+ * @return {Tensor}
+ */
+export function convTranspose2d(input, filter, {padding = [0, 0, 0, 0],
```

Comment:
[minor] Propose wrapping `padding` consistently to the same line as the others (strides, groups).

Reply:
Updated by new commits, thanks.